scripts/parser-tests/flow/allowlist.txt

```diff
@@ -14,9 +15,23 @@ class_properties/migrated_0003.js
 class_properties/migrated_0008.js
 class_properties/migrated_0021.js
 class_properties/migrated_0026.js
+comment_interning/class_method.js
+comment_interning/class_property.js
+comment_interning/declare_export_declaration.js
+comment_interning/declare_interface.js
+comment_interning/declare_type_alias.js
+comment_interning/decorator.js
+comment_interning/function_declaration.js
+comment_interning/import.js
+comment_interning/interface.js
+comment_interning/object_type.js
+comment_interning/remove_type_trailing_comments.js
+comment_interning/super.js
+comment_interning/type_alias.js
```

Comment:
Babel is throwing on these cases because of duplicate class declaration `class C`. The Flow parser is tolerant on duplicate binding, which are throw later by the type checker.